lib/browser/default-menu.js

```diff
@@ -0,0 +1,174 @@
+'use strict'
+
+const { shell, Menu } = require('electron')
+const isMac = process.platform === 'darwin'
+
+module.exports = () => {
+  const appMenu = {
+    label: 'Electron',
+    submenu: [
+      {
+        role: 'about'
+      },
+      {
+        type: 'separator'
+      },
+      {
+        role: 'services'
+      },
+      {
+        type: 'separator'
+      },
+      {
+        role: 'hide'
+      },
+      {
+        role: 'hideothers'
+      },
+      {
+        role: 'unhide'
+      },
+      {
+        type: 'separator'
+      },
+      {
+        role: 'quit'
+      }
+    ]
+  }
+
+  const fileMenu = {
+    label: 'File',
+    submenu: [{
+      role: isMac ? 'close' : 'quit'
+    }]
+  }
+
+  const editMenu = {
+    label: 'Edit',
+    submenu: [
```

Comment:
Why not use the `editMenu` role here?

> editMenu - Whole default "Edit" menu (Undo, Copy, etc.).

Reply:
good point! used that + added roles for several other default menus (app, view)